README.md

```diff
@@ -1,6 +1,32 @@
 # Product-Portal
 
-## Installation
+## Project Details
+
+### How do we complete this project?
+
+#### Ver 0.1 :
+  - **General concept:** Create a static website using array of objects (30 items - will be made by xx and yy) (each json item: title,price,link,catgeory…)
+    - navbar will include 6~ categories (which will be connected with the json file to right product category)
```

Comment:
Where are we getting these categories from?  How are deciding what these categories will be?  What happens if we click on a category?  Does it try to fetch all the items in the category from some api?